.github/workflows/update-script-in-vm.yaml

```diff
@@ -0,0 +1,22 @@
+name: Update Script in VM
+on:
+    push:
+      branches:
+        - 'master'
+      paths:
+        - '$GITHUB_WORKSPACE/.github/scripts/playground-cleanup.sh'
+jobs:
+    update_script:
+        runs-on: ubuntu-latest
+    
+        steps:
+        - name: Update Script in VM
+          uses: appleboy/ssh-action@master
+          with:
+            host: 147.75.47.9
+            username: root
+            key: ${{ secrets.METAL_SSH_KEY }}
+            script: |
+                cd
```

Comment:
Can you remove this `cd` ?

Reply:
I was trying to be conscious so that it gets into root dir will remove it